test/iron-dropdown.html

```diff
@@ -146,11 +171,14 @@
           dropdown = fixture('NonLockingDropdown');
         });
 
-        test('can be disabled with `allowOutsideScroll`', function() {
+        test('can be disabled with `allowOutsideScroll`', function(done) {
           dropdown.open();
 
-          expect(Polymer.IronDropdownScrollManager.elementIsScrollLocked(document.body))
-            .to.be.equal(false);
+          Polymer.Base.async(function() {
```

Comment:
The `scrolLock` element is updated in `_renderOpened` which I think is a tick after `_openedChanged`, and the test fails horribly without it